homeassistant/core.py

```diff
@@ -801,21 +801,27 @@ def async_run_hass_job(
     @overload
     @callback
     def async_run_job(
```

Comment:
Ideally we merge https://github.com/home-assistant/core/pull/113011 and than we don't need changes to `async_run_job`